src/styles/_card.scss

```diff
@@ -0,0 +1,102 @@
+@import "./variables.scss";
+
+p,
+a {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  border-radius: 5px;
+  border: 1px #f3f3f3 solid;
+  color: rgb(6, 11, 53);
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  padding-left: 16px;
+  padding-right: 16px;
+  width: 200px;
+
+  &__img {
+    display: block;
+    box-sizing: content-box;
+    padding: 32px 3px 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: $font-color-light;
+    font-size: $font-small;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    font-size: $font-small;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__buy-btn {
+    background: $color-blue;
+    border: 1px $color-blue solid;
+    border-radius: 5px;
+    color: white;
+    display: inline-block;
+    font-size: 14px;
+    font-weight: 700;
+    height: 40px;
+    line-height: 40px;
+    margin-bottom: 16px;
+    text-align: center;
+    text-decoration: none;
+    width: 166px;
+  }
+
+  &:hover {
+    background-color: white;
+    border: 1px solid $color-blue;
+    color: $color-blue;
+    transform: scale(120%);
+    transition-duration: 300ms;
```

Comment:
Transition property should be added for the default state of the element, not hover, so it works both ways correctly
![transition](https://github.com/mate-academy/layout_catalog/assets/96409563/c1f6d237-fef0-4fef-81c5-e7ac4fdf8b52)